cli/src/main/java/org/jboss/as/cli/impl/aesh/cmd/security/auth/AbstractEnableAuthenticationCommand.java

```diff
@@ -53,16 +44,11 @@
 import org.jboss.dmr.ModelNode;
 import org.wildfly.core.cli.command.DMRCommand;
 import org.wildfly.core.cli.command.aesh.CLICommandInvocation;
-import static org.jboss.as.cli.impl.aesh.cmd.security.SecurityCommand.OPT_NEW_SECURITY_REALM_NAME;
 import org.jboss.as.cli.impl.aesh.cmd.security.model.ExistingPropertiesRealmConfiguration;
-import static org.jboss.as.cli.impl.aesh.cmd.security.SecurityCommand.OPT_FILE_SYSTEM_REALM_NAME;
-import static org.jboss.as.cli.impl.aesh.cmd.security.SecurityCommand.OPT_PROPERTIES_REALM_NAME;
-import static org.jboss.as.cli.impl.aesh.cmd.security.SecurityCommand.OPT_EXPOSED_REALM;
-import static org.jboss.as.cli.impl.aesh.cmd.security.SecurityCommand.OPT_KEY_STORE_NAME;
-import static org.jboss.as.cli.impl.aesh.cmd.security.SecurityCommand.OPT_KEY_STORE_REALM_NAME;
-import static org.jboss.as.cli.impl.aesh.cmd.security.SecurityCommand.OPT_ROLES;
 import org.jboss.as.cli.impl.aesh.cmd.security.model.ExistingKeyStoreConfiguration;
 
+import static org.jboss.as.cli.impl.aesh.cmd.security.SecurityCommand.*;
```

Comment:
Please move these back to the top and expand back so they are fully qualified.

Reply:
Is this ok? The imports in the original are kinda all over the place.

Reply:
They do look a bit disorganised to begin with but all the static imports should be first in the import list so I think they should still move up.

Reply:
done